qa/workunits/rbd/cli_migration.sh

```diff
@@ -124,6 +141,125 @@ EOF
     remove_image "${dest_image}"
 }
 
+test_import_qcow_format() {
+    case "$(lsb_release --id --short)" in
+    RedHatEnterpriseWorkstation|RedHatEnterpriseServer|RedHatEnterprise|CentOS)
+        # QCOW format not included in EL variants
+        return
```

Comment:
nit: may be just return from `test_import_qcow_format` if `qemu-img convert -f raw -O qcow` failed below? So we don't need to explicitly check the release for the list which might become wrong in the future.

Reply:
Tweaked